android/gradle/dependencies.gradle

```diff
@@ -98,8 +98,8 @@ def androidx = [
 
 def test = [
     junit: "junit:junit:4.12",
-    mockito: "org.mockito:mockito-core:2.27.0",
-    mockitoKotlin: "com.nhaarman.mockitokotlin2:mockito-kotlin:2.2.0",
+    mockito: "org.mockito:mockito-core:4.6.1",
+    mockitoKotlin: "org.mockito.kotlin:mockito-kotlin:4.0.0",
```

Comment:
Do we need to keep these two versions in lockstep?

Reply:
No, mockito-kotlin do not follow the main mockito artifacts ([link](https://github.com/mockito/mockito-kotlin/tags))

Reply:
Currently `mockito-kotlin` based on Mockito 3.12.4. Starting from Mockito 4+, methods which were deprecated in 3+ was removed, so better to also bump-up `mockito-core` for using latest API in codebase. It will make future bump-up easier.